save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/InitI18n.kt

```diff
@@ -20,7 +20,35 @@ fun initI18n(isDebug: Boolean = false, interpolationEscapeValue: Boolean = false
         var resources = {
             en: {
                 translation: {
-                    "Notifications": "Notifications"
+                    "Notifications": "Notifications",
```

Comment:
`Notifications` translation here is kinda legacy.

It is better to move your translations to `Translations.kt` to a new `const val` - we prefer to split those translations into namespaces (split by view) - it will help us in later work.

After you created a new json in `Translation.kt`, add it into the main `resources` json of `InitI18n.kt`

Reply:
Got it